src/actions/PetitionActions.js

```diff
@@ -54,6 +60,12 @@ export function submittingPetition () {
 
 export function createPetition (petition, dispatch) {
   dispatch(submittingPetition());
+
+  // Wrap the URLs in an object to fit the API needs.
+  petition.links = petition.links
+    .filter((link) => link)
+    .map((link) => ({url: link}));
```

Comment:
@mattberridge 

This is the easiest way to do it, but also the recommended way by redux-form owner. Took me a while to find documentation on it.